### PR TITLE
Add Rome recommendations

### PR DESCRIPTION
**Pull request by matteocellucci**
This PR closes issue #79.
### Adding restaraunts and activities
- Fav foods?
- Do you like to know some gorgeous historical monuments?

@githubteacher

### CODE REVIEW

File: matteocellucci/matteocellucci-rome.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Rome :heart:
+
+#### Restaurants
+
+#### Things to do
```
Comment (GMartinez-Sisti) [inline matteocellucci/matteocellucci-rome.md]: Been there when I was a kid. Loved it.

**Comment by smanthena** [comment]
Any suggestions on eating places?

**Comment by matteocellucci** [comment]
@smanthena What type of food do you like?

File: matteocellucci/matteocellucci-rome.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Rome :heart:
```
Comment (ghost) [inline matteocellucci/matteocellucci-rome.md]: Rome Italy?   what is your favorite food which is not pasta?

Comment (matteocellucci) [inline matteocellucci/matteocellucci-rome.md]: @moyomendoza Yes, Italy 🇮🇹! Is this a trap for pizza 🍕? Anyway I really like a soup with lentil... but there is a little bit of pasta inside 😆

**Comment by RameshKatkam** [comment]
Historical Place. 300 Movie :) @matteocellucci

**Pull request merged**